relayer/README.md

```diff
@@ -309,6 +309,10 @@ The relayer is configured via a JSON file, the path to which is passed in via th
 
   - The AWS region in which the KMS key is located. Required if `kms-key-id` is provided.
 
+  `"block-acceptance-timeout-seconds": uint64`
```

Comment:
Would a more accurate name for this be `tx-inclusion-timeout-seconds`?

I know we already had an existing variable named `defaultBlockAcceptanceTimeout`, but I this it's a bit of a misnomer.